tools/android_lint/bin/main.dart

```diff
@@ -154,11 +159,34 @@ ArgParser setupOptions() {
   return argParser;
 }
 
+/// On macOS, we can try to find Java 1.8.
+///
+/// Otherwise, default to whatever JAVA_HOME is already.
+Future<String> getJavaHome() async {
+  if (Platform.isMacOS) {
+    final ProcessResult result = await processManager.run(
+      <String>['/usr/libexec/java_home', '-v', '1.8'],
+    );
+    return result.stdout.trim();
+  }
+  return Platform.environment['JAVA_HOME'];
```

Comment:
Looks like this is failing in CI:

`Invalid argument(s): Environment key or value is not a string: (JAVA_HOME, null)`